src/client/src/components/SelectableCard/index.tsx

```diff
@@ -121,8 +126,16 @@ const SelectableCard = ({
           onKeyUp={keyUpHandler}
         >
           <FormattedMessage
-            id="selectableCard.details"
-            defaultMessage="Details"
+            id={
+              isPagesSelection
+                ? detailsConfig.preview.id
+                : detailsConfig.learnMore.id
+            }
+            defaultMessage={
+              isPagesSelection
+                ? detailsConfig.preview.default
+                : detailsConfig.learnMore.default
+            }
           />
```

Comment:
Maybe we can change it to check condition only once and get rid of detailsConfig?
```isPagesSelection ? <FormattedMessage id="selectableCard.preview" ...> : <FormattedMessage id="selectableCard.details" ...>```

Reply:
I was thinking of doing that initially but I thought that it would be more organized to store it config object especially in case we needed to add a third option later

Reply:
Yup, if we consider adding more options, it makes sense :)